src/main/java/com/flink/connector/BiqQueryReadSession.java

```diff
@@ -0,0 +1,40 @@
+package com.flink.connector;
+
+import org.apache.flink.util.Preconditions;
+
+import com.google.cloud.bigquery.storage.v1.BigQueryReadClient;
+import com.google.cloud.bigquery.storage.v1.CreateReadSessionRequest;
+import com.google.cloud.bigquery.storage.v1.DataFormat;
+import com.google.cloud.bigquery.storage.v1.ReadSession;
+import com.google.cloud.bigquery.storage.v1.ReadSession.TableModifiers;
+import com.google.protobuf.Timestamp;
+
+public class BiqQueryReadSession {
```

Comment:
Can be removed, it duplicates ReadSessionCreator

Reply:
Compared the functionalities in both the files, BigQueryReadSession class is created for getting read session from the ReadSessionCreator class. The functionalities in "BigQueryReadSession " is to create config properties that need to be passed to "ReadSessionCreator " class.